hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/log/LogLevel.java

```diff
@@ -349,7 +348,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response
         }
 
         if (GenericsUtil.isLog4jLogger(logName)) {
```

Comment:
yes unfortunately, we will still have some log4j references outside of hadoop-logging, moving all references is extremely difficult.
i am planning to get upgraded to log4j2 with this change and then eventually we can try moving all usages to hadoop-logging. With this PR, we have almost 95% of log4j references moved to hadoop-logging.

Reply:
Thanks @Apache9 for raising this point btw.

let me paste my [comment](https://issues.apache.org/jira/browse/HADOOP-16206?focusedCommentId=17702846&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17702846) from parent jira, the jira has too many comments so can be easily ignored :)


_FYI, we use some of the core log4j APIs specifically meant to be used for properties based config and log4j2 has this support removed as they were meant to be private usage. Moreover, we also programmatically set monitor interval for dynamic log4j file changes in httpfs server. This is also no longer supported as that is also meant to be kept in config only (properties, xml, json, yaml etc)._

_I have started this thread with log4j dev/user group for the same https://lists.apache.org/thread/4l7oyk84jpj6br0sn4ymofdcbgfxmtqp_

_So far, the only recommended solution provided by them is to configure monitor interval in log4j properties only (i.e. user provided custom log4j properties, in case of httpfs Server), and use Configurator API for the properties file specific usage we have._